packages/lightning-lsp-common/scripts/build.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const shell = require('shelljs');
+
+// Copy static assets
+if (shell.exec('yarn copy-static-assets"').code !== 0) {
```

Comment:
Same as the comments for the script in aura-language-server, let's move the logic in `copy-static-assets.ts` here so we can remove the use of ts-node and that ts file

Reply:
Yeah, i was about to do that, as ts-node is only used to run that single file